Client/Client.Core/Client.Core.Static/Client.hpp

```diff
@@ -64,160 +61,148 @@ class Client
 
     /**
     * @brief Loop that handle incoming messages.
-    * @details Every type of message have specific function handler, \
-    *   which in the process of processing the incoming message from the server goes \
-    *   in a loop for further processing of the response from the client.
+    * @details Handler function to process the server response message \
+    *   for the subsequent response from the client.
+    *   (You can see the message types in the Message header file).
```

Comment:
The central method in which most of the work of the client kernel takes place. 
The function is an infinite loop that waits for messages to appear in the message queue.
Messages are divided into types and a different handler is called for each type. 
In the case of a non-registered type, an issue is logged and the work continues.